### PR TITLE
Bypass template processing 2.3

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/18218

### CODE REVIEW

**Review by gparvin — COMMENTED** [review]
We didn't have hub templates support in ACM 2.3.  We did have the managed cluster templates support.  I'll have to let @ckandag comment on whether the annotation to disable the template processing was backported to 2.3.

**Comment by ckandag** [comment]
yes, it was backported to 2.3

> We didn't have hub templates support in ACM 2.3. We did have the managed cluster templates support. I'll have to let @ckandag comment on whether the annotation to disable the template processing was backported to 2.3.

yes, it was backported to 2.3, while the hub templates are not in 2.4,  managedcluster templates can still be disabled by setting the annotation on the configuration policy definition.

but looks like you have ported entire hub template section as well which needs to be fixed. pls see inline comments

File: governance/custom_template.adoc [hunk 1]
```diff
@@ -361,3 +368,72 @@ spec:
         {{ (fromConfigMap "site-config" "site1" "enabled")  | toBool }}
 ----
 
+[#managed-templates]
```
Comment (ckandag) [inline governance/custom_template.adoc]: this entire section needs to be removed for 2.3. as we do not support hub templates in 2.3, only the "bypassing" section should be included for 2.3

File: governance/custom_template.adoc [hunk 2]
```diff
@@ -361,3 +368,72 @@ spec:
...
+| You can reference any resource on the cluster.
+
+| Processing errors
+| Errors from the hub cluster templates are displayed as violations on the managed clusters the policy applies to.
+| Errors from the managed cluster templates are displayed as violations on the specific target cluster where the violation occurred.
+|===
+
+
+[#bypass-template-processing]
+=== Bypass template processing
```
Comment (ckandag) [inline governance/custom_template.adoc:423]: this section is the only thing that needs to be added to 2.3

File: governance/custom_template.adoc [hunk 3]
```diff
@@ -361,3 +368,72 @@ spec:
...
+
+You might create a policy that contains a template that is not intended to be processed by {product-title-short}. By default, {product-title-short} processses all templates. 
+
+To bypass template processing for your hub cluster, you must change `{{ template content }}` to `{{ '{{ template content }}' }}`.
```
Comment (ckandag) [inline governance/custom_template.adoc]: to bypass policy template processing, you must change ....

no need to say "for your hub cluster". - it actually isnt just for hub cluster isnt for every /any cluster

**Review by ckandag — CHANGES_REQUESTED** [review]
pls see inline comments

**Review by ckandag — APPROVED** [review]

**Comment by dockerymick** [comment]
Thanks for the approval @ckandag !

**Pull request merged**